include/beast/http/detail/rfc7230.hpp

```diff
@@ -318,20 +319,23 @@ increment()
     {
         ++it;
         if(it == end)
-            return err();
+            break;
         if(! detail::is_tchar(*it))
             break;
     }
     auto const p1 = it;
+    v.first = { &*p0, static_cast<std::size_t>(p1 - p0) };
     detail::skip_ows(it, end);
     if(it == end)
-        return err();
+        return;
```

Comment:
I see what you're going for but I think `{err(); return;}` is clearer.
Considering snippet `err(); return` tells me both functions return void. Where `return err();` does not. IMHO